### PR TITLE
HCIDOCS-139: Fix nohup command in network config for IPI/bare metal installation

### PR DESCRIPTION
**Pull request by johnwilkins**
Added static IP example.

Fixes: [HCIDOCS-139](https://issues.redhat.com//browse/HCIDOCS-139)

See https://issues.redhat.com/browse/HCIDOCS-139 for additional details.

Preview URL:

For release(s): 4.12-4.18
QE Review: 

- [ ] QE has approved this change. 

Signed-off-by:  <jowilkin@redhat.com>

### CODE REVIEW

File: modules/ipi-install-configuring-networking.adoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ You can also configure networking from the web console.
 
 .Procedure
 
-. Export the bare-metal network NIC name:
+. Export the bare-metal network NIC name by running the following command:
```
[the remark this answers is not in the document]
Comment (johnwilkins) [reply modules/ipi-install-configuring-networking.adoc:20]: I think this is wrong. "network" is a noun.

File: modules/ipi-install-configuring-networking.adoc [hunk 2]
```diff
@@ -69,34 +89,35 @@ $ sudo nohup bash -c "
 +
 [NOTE]
 ====
-The ssh connection might disconnect after executing these steps.
+The SSH connection might disconnect after executing these steps.
 
-The IPv6 address can be any address as long as it is not routable via the bare-metal network.
+The IPv6 address can be any address provided it is not routable through the bare-metal network.
```
[the remark this answers is not in the document]
Comment (johnwilkins) [reply modules/ipi-install-configuring-networking.adoc:94]: I think this is wrong. "network" is a noun.

**Comment by mkowalski** [comment]
Looks okay. If someone verified that copy-paste of all the commands included is giving expected result, it's okay to merge

File: modules/ipi-install-configuring-networking.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (mburke5678) [reply modules/ipi-install-configuring-networking.adoc:20]: I agree in this case.

File: modules/ipi-install-configuring-networking.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (mburke5678) [reply modules/ipi-install-configuring-networking.adoc:94]: > any address provided

I read this as _any address that you provided_... Maybe:
```suggestion
The IPv6 address can be any address that is not routable through the bare-metal network.
```

File: modules/ipi-install-configuring-networking.adoc [hunk 3]
```diff
@@ -69,34 +89,35 @@ $ sudo nohup bash -c "
...
-. `ssh` back into the `provisioner` node (if required):
+. SSH back into the `provisioner` node (if required) by running the following command:
 +
 [source,terminal]
 ----
 # ssh kni@provisioner.<cluster-name>.<domain>
 ----
 
-. Verify the connection bridges have been properly created:
+. Verify the connection bridges have been properly created by running the following command:
```
Comment (mburke5678) [inline modules/ipi-install-configuring-networking.adoc:113]:
```suggestion
. Verify that the connection bridges have been properly created by running the following command:
```

**Review by skrthomas — COMMENTED** [review]
merge review /lgtm

**Pull request merged**